styles/index.css

```
@@ -32,17 +32,25 @@ h1 {
   padding: 15px;
 }
 
-.card button {
+.card .card-answer-btn {
```

This should work as well, wouldn't it?

```suggestion
.card-answer-btn {
```